src/pages/SubscriptionInventoryPage/SubscriptionInventoryPage.tsx

```diff
@@ -38,15 +41,31 @@ const SubscriptionInventoryPage: FunctionComponent = () => {
             <StackItem>
               <GettingStartedCard />
             </StackItem>
+            <StackItem>
+              <>
```

Comment:
Lines 45-57 look exactly the same as 61-69, just with a different component. It may be worth creating some kind of component to do all of this rendering logic for whatever children are provided. This may be elsewhere in the application too that could be abstracted away so that we aren't repeating ourselves. Rough ex

```suggestion
              <Fetchable data={statusCardData} component={StatusCountCards}>
```